packages/azure-kusto-ingest/index.html

```diff
@@ -0,0 +1,6 @@
+<!DOCTYPE html>
```

Comment:
Why is this here? it doesn't work against exampleBrowser (unless you build and rename the file manually)

Maybe it should be a full seperate project that shows the whole process?